assets/css/styles.css

```
@@ -341,33 +348,43 @@ button.sponsor-us {
   border-width: 1px;
   border-style: solid;
   border-color: #7fffd6;
-  transform: translate3d(0, 50px, 0) rotate(0.0001deg);
-  -webkit-transform: translate3d(0, 50px, 0) rotate(0.0001deg);
+  transform: rotate(0.0001deg);
+  -webkit-transform: rotate(0.0001deg);
   -webkit-backface-visibility: hidden;
   backface-visibility: hidden;
   -webkit-perspective: 1000;
   perspective: 1000;
   animation: fadeIn .5s both;
   animation-delay: .5s;
+  margin-top: 50px;
 }
 
 button.sponsor-us:hover {
   opacity: .95;
   box-shadow: rgba(71, 53, 13, 0.3) 0px 4px 8px 0px;
   transform: translateY(45px) rotate(0.0001deg);
-  -webkit-transform: translateY(45px) rotate(0.0001deg);
+  -webkit-transform: translateY(-5px) rotate(0.0001deg);
 }
 
 .landing-picture {
-  padding-right: 9vw;
+  display: flex;
+  align-items: flex-end;
+  height: 100%;
 }
 
 img.tower {
   width: 100%;
   height: auto;
-  -webkit-transform: scale(2.8);
-  -ms-transform: scale(2.8);
-  transform: scale(2.8);
+  -webkit-transform: scale(1.9);
+  -ms-transform: scale(1.9);
+  transform: scale(1.9);
+
+  -webkit-transform-origin : 70% 90%;
+  -moz-transform-origin : 70% 90%;
+  -o-transform-origin : 70% 90%;
+  -ms-transform-origin : 70% 90%;
+  transform-origin : 70% 90%;
```

```suggestion
  transform-origin : 70% 60%;
```

Changed:
`70% 60%`'s position makes more sense, otherwise it will stuck at the top right corner at 1440px

---

Refer to the review, otherwise resolved.